suitcase.py

```diff
@@ -39,34 +40,30 @@ def export(headers, filename):
                 desc_group.create_dataset('event_times', event_times)
                 data_group = desc_group.create_group('data')
                 ts_group = desc_group.create_group('timestamps')
+                [fill_event(e) for e in events]
                 for key, value in data_keys.items():
+                    print('data key = %s' % key)
                     timestamps = [e['timestamps'][key] for e in events]
                     ts_group.create_dataset(key, data=timestamps)
-                    for event in events:
-                        fill_event(event)  # fill any external data, in place
                     data = [e['data'][key] for e in events]
                     dataset = data_group.create_dataset(key, data=data)
                     # Put contents of this data key (source, etc.)
                     # into an attribute on the associated data set.
-                    value['data_broker_shape'] = value.pop('shape')
-                    value['data_broker_dtype'] = value.pop('dtype')
-                    print(value)
-                    _safe_attrs_assignment(dataset, value)
+                    # value['data_broker_shape'] = value.pop('shape')
```

Comment:
bump

@ericdill Can you restore these deleted lines and comment them out instead of deleting them? Then we can merge and return to this later when there is demand/bandwidth.